src/mixins/date-mixin.ts

```diff
@@ -121,15 +121,8 @@ export default class DateMixin extends Mixins(CommonMixin) {
     // safety checks
     if (!dateStr) return null
     const date = this.createUtcDateFromStr(dateStr)
-    if (!date) return null
 
-    let result = date.toLocaleDateString('en-CA', {
-      timeZone: 'America/Vancouver',
-      weekday: showWeekday ? 'long' : undefined, // Thursday or nothing
-      month: longMonth ? 'long' : 'short', // December or Dec.
-      day: 'numeric', // 31
-      year: 'numeric' // 2020
-    })
+    let result = this.dateToPacificDate(date, true)
```

Comment:
Maybe just copy in `yyyyMmDdToPacificDate()` from my PR (or main branch) so that your rebase becomes a trivial merge.

Reply:
done